### PR TITLE
DB-8085 Can also use ALIAS instead of SYNONYM

### PR DESCRIPTION
**Pull request by wb14123**

### CODE REVIEW

File: splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/CreateAliasIT.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    public void testCreateSynonym() throws  Exception {
+        methodWatcher.executeUpdate("create synonym t2 for t1");
+        methodWatcher.executeUpdate("drop synonym t2");
+    }
+
+    @Test
+    public void testCreateAlias() throws  Exception {
+        methodWatcher.executeUpdate("create alias t2 for t1");
+        methodWatcher.executeUpdate("drop alias t2");
+
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/CreateAliasIT.java]: Can you add a few tests selecting from a synonym, etc?

Comment (wb14123) [reply splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/CreateAliasIT.java]: Added the test about select. While adding the test, found a bug about create synonym. Created the ticket DB-8854.

Comment (dgomezferro) [reply splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/CreateAliasIT.java]: Thanks @wb14123 !

**Comment by OlegMazurov** [comment]
Will there be a PR for branch-3.0?

**Comment by wb14123** [comment]
Created 3.0 PR #2993

**Pull request merged**